core/commands/swarm.go

```diff
@@ -599,7 +599,7 @@ var swarmAddrsLocalCmd = &cmds.Command{
 		for _, addr := range maddrs {
 			saddr := addr.String()
 			if showid {
-				saddr = path.Join(saddr, p2pProtocolName, self.ID().Pretty())
+				saddr = path.Join(saddr, p2pProtocolName, self.ID())
```

Comment:
@Icarus9913 can you fix the instances where you removed `.Pretty` but didn't replace them? Code doesn't compile.

Reply:
oh my bad. I missed it and wonder why CI failed.
Solved.